### DISCUSSION TITLE
The value output by `process.cwd()` is different during build and run, causing the file to not be read

### DISCUSSION THREAD

**Discussion by mikeguo99** [vercel/next.js#67844] [App Router] [answered]
I found many responses suggesting that when deploying on Vercel, files should be read using `path.join(process.cwd(), "your.file.path")`. This works fine locally and even worked previously on Vercel. However, today I discovered that deploying a new project on Vercel with this method results in an error:

```
Error: ENOENT: no such file or directory, open '/var/task/your/file/path' ...
```

I then added logs and observed that the output of `process.cwd()` differs between build and runtime, specifically:
  - `/vercel/path0` when building;
  - `/var/task` when running.

I suspect that Vercel has modified the build output directory in a recent version, and now my source code's `/data` directory is not being copied to `/var/task`.

Therefore, I want to know if Vercel has an official solution to this issue.

**Comment by mikeguo99** [comment, marked as answer]
oh I solved it by specifying `experimental.outputFileTracingIncludes` to `next.config.mjs`. Like bellow:
```mjs
const nextConfig = {
  experimental: {
    outputFileTracingIncludes: {
      "/": ["./data/**/*"],
    },
  },
};

export default nextConfig;
```

**Reply by JRH89** [reply]
worked for me.
